.prow.yaml

```diff
@@ -17,15 +17,10 @@ presubmits:
   - name: e2e
     annotations:
     labels:
-    run_if_changed: 'E2E_instance.txt'
+    run_if_changed: '^e2e/'
     skip_report: false
     decorate: true
     cluster: default
-    extra_refs:
```

Comment:
why are we removing this?

Reply:
That action is redundant, we're cloning the repo during the execution of `gce_run.sh` script.